release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -117,7 +117,14 @@ The following Operators are supported for {product-title} on ARM:
 [id="ocp-4-10-ibm-cloud-installation"]
 ==== Installing a cluster on IBM Cloud using installer-provisioned infrastructure (Technology Preview)
 
-{product-title} 4.10 introduces support for installing a cluster on IBM Cloud using installer-provisioned infrastructure.
+{product-title} 4.10 introduces support for installing a cluster on IBM Cloud using installer-provisioned infrastructure (IPI).
+
+The following limitations apply for IBM Cloud using IPI:
+
+* Networks must be created ahead of time (Bring Your Own Network). Deploying IBM Cloud using IPI on a previous existing network is not supported.
```

Comment:
I think you can delete: "Networks must be created ahead of time (Bring Your Own Network)" and just go with "Deploying IBM Cloud using IPI on a previous existing network is not supported." 
To me, creating a network ahead of time sounds like Bring your own network, but I am be misreading it.

Reply:
@sjstout I borrowed that phrasing from [elsewhere in the release notes](https://docs.openshift.com/container-platform/4.10/release_notes/ocp-4-10-release-notes.html#ocp-4-10-openstack-dcn-for-enterprise). I can remove it. Seems that these are the only places in the docs where we mention Bring Your Own Network.

Reply:
Yes, this `Networks must be created ahead of time (Bring Your Own Network).` needs to be removed